docs/data/joy/getting-started/installation/installation.md

```diff
@@ -72,6 +72,6 @@ To install Inter through the Google Web Fonts CDN, add the following code inside
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 <link
   rel="stylesheet"
-  href="https://fonts.googleapis.com/css?family=Inter:wght@300;400;500;600;700&display=swap"
+  href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
 />
```

Comment:
This was broken, variables fonts are only in v2 https://developers.google.com/fonts/docs/css2